noether_tpp/include/noether_tpp/tool_path_modifiers/blending_modifiers.h

```diff
@@ -0,0 +1,58 @@
+﻿#pragma once
+
+#include <noether_tpp/core/tool_path_modifier.h>
+
+namespace noether
+{
+/**
+ * @brief Transforms the waypoints to correspond with the center of the ginding tool so that the edge
+ * of the tool is in contact with the media
+*/
+class AngledOrientationModifier : public ToolPathModifier
```

Comment:
We should probably come up with a more specific name than this; maybe something like `ToolDragOrientationToolPathModifier`